packages/suite-web-landing/pages/_document.tsx

```diff
@@ -59,9 +59,7 @@ export default class MyDocument extends Document {
                     />
                     <meta
                         property="og:image"
-                        content={`${URLS.SUITE_URL}${resolveStaticPath(
```

Comment:
you probably could still use resolveStaticPath just to be more future-proof, but should work anyway.

Reply:
Probably yes, we can't test it properly on sldev anyway so at least the `static` path would't be hardcoded. I'll change it back